src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -206,8 +206,26 @@ public void testEnumsAndAnnotations() throws Exception {
     @Test
     public void testNoJavadocs() throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(WriteTagCheck.class);
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        verify(checkConfig, getPath("InputWriteTagNoJavadoc.java"), expected);
+    }
+
+    @Test
+    public void testNoJavadocsWithRequiredTag() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(WriteTagCheck.class);
+        checkConfig.addProperty("tag", "@incomplete");
+        final String[] expected = {
+            "13: " + getCheckMessage(MSG_MISSING_TAG, "@incomplete"),
+        };
+        verify(checkConfig, getPath("InputWriteTagNoJavadoc.java"), expected);
```

Comment:
please make a separate test inputs for each test

Reply:
Added separate files for the TCs